Header header

float32 X_Accel
float32 Y_Accel
float32 Z_Accel

float32 X_Accel_noG
float32 Y_Accel_noG
float32 Z_Accel_noG

float32 X_AngVel
float32 Y_AngVel
float32 Z_AngVel

float32 A_AngPos
float32 B_AngPos
float32 C_AngPos
float32 D_AngPos

float32 X_Mag
float32 Y_Mag
float32 Z_Mag
